sites/main-page-title/main-page-title-en.md

```diff
@@ -0,0 +1,145 @@
+---
+altLangPage: "main-page-title-fr.html"
+breadcrumbs:
+  - title: "GCWeb home"
+    link": "https://wet-boew.github.io/GCWeb/index-en.html"
+dateModified: "2023-07-24"
+description: "Documentation on the default H1 use as the main title of a page with short bold red underline."
+language: "en"
+pageclass: "wb-prettify all-pre"
+title: "Main page title"
+---
```

Comment:
You should use our special page documentation template and remove all the body content unless if they are additional information not included in our default template.

See: 
* https://github.com/wet-boew/GCWeb/blob/master/sites/feedback/feedback-docs-en.html
* https://github.com/wet-boew/GCWeb/blob/master/sites/feedback/feedback-docs-fr.html


```suggestion
---
altLangPage: "main-page-title-fr.html"
breadcrumbs:
  - title: "GCWeb home"
    link": "https://wet-boew.github.io/GCWeb/index-en.html"
dateModified: "2023-07-24"
description: "Documentation on the default H1 use as the main title of a page with short bold red underline."
language: "en"
pageclass: "wb-prettify all-pre"
title: "Main page title"
layout: documentation
index_json: index.json-ld
---
```